cmds/rm/rm.go

```diff
@@ -69,10 +64,10 @@ func rm(files []string) error {
 			return err
```

Comment:
Shouldn't this be return nil? They don't want to remove it. There's no error here?